src/Apps/Order/Routes/__tests__/Shipping.jest.tsx

```diff
@@ -1121,26 +1118,33 @@ describe("Shipping", () => {
               "SetShippingMutation"
             )
 
-            expect(
-              page.find(ShippingRoute).state().shippingQuotes
-            ).toHaveLength(5)
+            selectShippingQuoteSuccess.commerceSelectShippingOption.orderOrError.order.lineItems.edges[0].node.shippingQuoteOptions.edges.forEach(
```

Comment:
Sorry for the :ceiling-cat here 😅
I guess this assertion is not proper, we're comparing the mocked value from selectShippingQuoteSuccess with the values from the exported shippingQuoteDisplayNames that have nothing to do with the component testing itself, going to refactor it this week to be more assertive in regards to what we expect from the component and will tag you @AlicanAkyuz